tests/commands/ddtrace_run_appsec.py

```diff
@@ -0,0 +1,8 @@
+from ddtrace.appsec import _mgmt
+
+
+if __name__ == "__main__":
+    if _mgmt.protections:
```

Comment:
Does it maybe make sense to have an internal API for this on the manager class?

```python
class Management:
    # ...
    @property
    def _loaded(self) -> bool:
        # Return whether appsec has successfully loaded or not
        return bool(self.protections)
```

(especially if protections ends up being private)

Reply:
I added a `enabled` property on the management class!